com/williamfiset/algorithms/sorting/RadixSort.java

```diff
@@ -1,12 +1,73 @@
-/**
- * NOTE: This file is still under development
- **/
-package com.williamfiset.algorithms.sorting;
-
-public class RadixSort {
-
-  public static void radixSort(int[] ar) {
-    
-  }
-
-}
+
+public class radixSort
+{
+
+        static int[] radix(int []a)
+        {
+
+            // Counting sort is also the part of radix sort. So, you should have the knowledge about it.
+
+            int b[] = new int[10];
+            int sorted[] = new int[a.length];
+            int max = a[0];
+            int divisor = 1;
+
+            for(int i = 0; i < sorted.length; i++)  // copy contents of a[] so that the original array is not changed
+            {
+                sorted[i] = a[i];
+            }
+
+            for(int i = 0; i < sorted.length; i++)  // finding max to implement counting sort. (Range of elements)

```

Comment:
I have not yet looked at the logic behind the algorithm -- waiting until you add tests.